lib/Controller/OpenProjectAPIController.php

```diff
@@ -197,15 +198,17 @@ public function linkWorkPackageToFile(int $workpackageId, int $fileId, string $f
 
 		try {
 			$result = $this->openprojectAPIService->linkWorkPackageToFile(
-				$workpackageId,
-				$fileId,
-				$fileName,
+				$values,
 				$this->userId,
 			);
 		} catch (OpenprojectErrorException $e) {
 			return new DataResponse($e->getMessage(), Http::STATUS_BAD_REQUEST);
 		} catch (NotPermittedException | NotFoundException $e) {
 			return new DataResponse('file not found', Http::STATUS_NOT_FOUND);
+		} catch (InvalidArgumentException $e) {
+			return new DataResponse([
+				"error" => $e->getMessage()
+			], Http::STATUS_BAD_REQUEST);
 		} catch (Exception $e) {
```

Comment:
While you're at it can you refactor this to something like this? seems like we missed the `\`
```suggestion
		} catch (\Exception $e) {
```